NEWS.md

```diff
@@ -12,6 +12,21 @@
 were enhanced such that more than one summary variable can be derived, e.g.,
 `AVAL` as the sum and `ADT` as the maximum of the contributing records. (#1792)
 
+- The "joined" functions (`derive_vars_joined()`, `derive_var_joined_exist_flag()`,
```

Comment:
```suggestion
- The "joined" functions (`derive_vars_joined()`, `derive_var_joined_exist_flag()`,
```

Reply:
I think these quotes have caused us issues 
![image](https://github.com/pharmaverse/admiral/assets/10111024/b87d3462-4cbf-4748-af79-f51f750dc9a6)

Reply:
Are you sure?
There are many more double quotes in `NEWS.md`.